shared/reducers/login.js

```diff
@@ -37,8 +36,6 @@ const initialState: Constants.State = {
 }
 
 export default function(state: Constants.State = initialState, action: any): Constants.State {
-  let toMerge = null
```

Comment:
this is a bad pattern. if we changed a sub thing we'd encode the whole thing into immutable just to merge so we can do toObject at the end making every property everywhere thrash